utils/datetime/LocalDate.js

```diff
@@ -41,29 +41,22 @@ export class LocalDate {
      * Get an instance of this class.
      * This is the standard way to get an instance of this object from serialized server-side data.
      *
-     * @param {string} s - a valid date in 'YYYYMMDD' format.
+     * @param {string} s - a valid date in 'YYYY-MM-DD' or 'YYYYMMDD' format.
      * @returns {LocalDate}
      */
     static get(s) {
         if (isNil(s)) return s;
-        throwIf(!isString(s) || !LocalDate.fmtRegEx.test(s), 'LocalDate.get() requires a string of the form "YYYYMMDD"');
-
-        let {_instances} = this,
-            ret = _instances.get(s);
-
-        if (!ret) {
-            ret = new LocalDate(s);
-            _instances.set(s, ret);
-        }
-
-        return ret;
+        if (!isString(s)) this.throwInvalidArgument(s);
+        s = s.replaceAll('-', '');
```

Comment:
if here the code now took just the first 8 chars, it would open up easily converting many time strings to LocalDate.
Is there a safety/integrity use case where if a developer deliberately tries to convert a time string that has time or zone info to a LocalDate they should be blocked?

Reply:
You can use .from if you want that kind of parsing

Reply:
Sure, but isn't the point of this change to make it easier to feed values into LocalDate when the feeding is done by an intermediary component, like the column parser, which uses only .get?